FindOnPage.md

```diff
@@ -0,0 +1,498 @@
+[uuid(2604789D-9553-4246-8E21-B9C74EFAD04F), object, pointer_default(unique)]
+interface ICoreWebView2FindStartFindOperationCompletedHandler: IUnknown {
+    /// Param1 refers to the returned code when the find operation completes.
+    /// Param2 refers whether the find session successfully finished executing or not.    
+    HRESULT Invoke(HRESULT errorCode, BOOL status);
+}
+
+// Interface providing methods and properties for finding and navigating through text in the web view.
+[uuid(7C49A8AA-2A17-4846-8207-21D1520AABC0), object, pointer_default(unique)] 
+interface ICoreWebView2Find : IUnknown {
+ 
+    // Initiates a find using the specified configuration.
+    HRESULT StartFind([in] ICoreWebView2FindConfiguration* configuration, 
+                      ICoreWebView2FindOperationCompletedHandler* handler);
+
+    // Navigates to the next match in the document.
+    HRESULT FindNext();
+
+    // Navigates to the previous match in the document.
+    HRESULT FindPrevious();
+
+    // Stops the current 'Find' operation and hides the Find bar.
+    HRESULT StopFind();
+
+    // Registers an event handler for the FindCompleted event.
+    // This event is raised when a find operation completes, either by finding all matches, navigating to a match, or by being stopped.
+    // Parameter is the event handler to be added.
+    // Returns a token representing the added event handler. This token can be used to unregister the event handler.
+    HRESULT add_matchCount Changed(
+        [in] ICoreWebView2FindmatchCount ChangedEventHandler* eventHandler,
+        [out] EventRegistrationToken* token);
+    // Unregisters an event handler from the matchCount Changed event.
+    // Parameter is the token of the event handler to be removed, obtained during the registration of the event handler.
+    HRESULT remove_matchCount Changed([in] EventRegistrationToken token);
+
+    // Registers an event handler for the ActiveMatchIndexChanged event.
+    // This event is raised when the index of the currently active match changes.
+    // This can happen when the user navigates to a different match or when the active match is changed programmatically.
+    // Parameter is the event handler to be added.
+    // Returns a token representing the added event handler. This token can be used to unregister the event handler.
+    HRESULT add_ActiveMatchIndexChanged(
+        [in] ICoreWebView2FindActiveMatchIndexChangedEventHandler* eventHandler,
+        [out] EventRegistrationToken* token);
+    // Unregisters an event handler from the ActiveMatchIndexChanged event.
+    // parameter is the token of the event handler to be removed, obtained during the registration of the event handler.
+    HRESULT remove_ActiveMatchIndexChanged([in] EventRegistrationToken token);
+}
+```
+
+
+### Setting Up Find Configuration with MIDL3
+
+To represent the given C# examples in a manner consistent with the behavior demonstrated 
```

Comment:
You don't need this paragraph of text / section. Just the MIDL3 labeled as MIDL3 is good.